test/_references.ts

```diff
@@ -38,7 +38,7 @@
 /// <reference path="../node_modules/powerbi-visuals-utils-testutils/lib/index.d.ts"/>
```

Comment:
We don't need to test/_references.ts file anymore to configure typings. File should be removed.

Reply:
thanks!